ui/src/pages/backoffice/components/KeyValueTable/KeyValueTable.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import isEmpty from 'lodash/isEmpty';
+import { Table } from 'semantic-ui-react';
+
+export const KeyValueTable = ({ keyWidth, data, showEmpty }) => {
```

Comment:
there was something similar implemented already,  `MetadataTable` as I remember, maybe it makes sense to  use one of them, and remove the other